plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -607,18 +607,13 @@ private function get_setting_for_feature( string $feature_id, array $feature, bo
 			$disabled = true;
 			$desc_tip = __( 'WooCommerce Admin has been disabled', 'woocommerce' );
 		} elseif ( 'new_navigation' === $feature_id ) {
-			$needs_update = version_compare( get_bloginfo( 'version' ), '5.6', '<' );
-			if ( $needs_update && current_user_can( 'update_core' ) && current_user_can( 'update_php' ) ) {
-				$update_text = sprintf(
-				// translators: 1: line break tag, 2: open link to WordPress update link, 3: close link tag.
-					__( '%1$s %2$sUpdate WordPress to enable the new navigation%3$s', 'woocommerce' ),
-					'<br/>',
-					'<a href="' . self_admin_url( 'update-core.php' ) . '" target="_blank">',
-					'</a>'
-				);
-				$description .= $update_text;
-				$disabled     = true;
-			}
+			$update_text = sprintf(
+				// translators: 1: line break tag.
+				__( '%1$s The development of this feature is currently on hold.', 'woocommerce' ),
+				'<br/>'
+			);
+			$description .= $update_text;
+			$disabled     = true;
```

Comment:
One very minor thing here is that if the navigation was previously enabled, the checkbox still appears checked even though it's disabled.

Its grayed out and I don't have a strong opinion on this, but I *think* unchecking might feel slightly more correct.
<img width="792" alt="Screen Shot 2023-01-18 at 10 14 31 AM" src="https://user-images.githubusercontent.com/10561050/213261943-ff5a92c7-094e-4b23-adb5-ec0a6728be7d.png">

Reply:
I agree. Fixed.